src/Apps/Notifications/notificationsRoutes.ts

```diff
@@ -25,4 +33,18 @@ export const notificationsRoutes: AppRouteConfig[] = [
       }
     `,
   },
+  {
+    path: "/notifications/:id",
```

Comment:
**nitpick:**
```suggestion
    path: "/notification/:id",
```

Reply:
Good catch! I guess using plural nouns for collections is best practice, but you're right. In Force & Eigen, we use singular nouns for these kinds of routes. ([example](https://github.com/artsy/eigen/blob/c872a882bb1a374e2b5db1d4a12e404de3209a99/src/app/routes.ts#L171)). I will update the URL.

Reply:
I was also confused because we have unfortunately both patterns in some cases that's why I felt this should not be a blocker for us